lib/Subscriber/Behavior/Path/AutoNameSubscriber.php

```diff
@@ -199,6 +200,9 @@ private function getName(
 
         $name = $this->slugifier->slugify($title);
 
+        // jackrabbit can not handle node-names which contains a number followed by "e" e.g. 10e
+        $name = preg_replace('(([\.%-]?\d+)([eE]))', '$1-$2', $name);
```

Comment:
The first part of the regex `[\.%-]?` could be removed now, since it matches everytime. The `?` modifier says that it can occur, but doesn't have to. If we really want to make this happen only when there is a `.`, `%`, `-` or nothing in front of it the regex gets more complicated, and I don't know if it is worth it 😕